CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+## Target branch
+
+Pull requests should target the `main` branch. If the change is relevant for a hotfix release, make a note in the PR description so it can be cherry picked after merge.
+
+## Adding to CHANGELOG
+
+All changes focused on users of the released builds should be documented by writing a new entry to [CHANGELOG.md](CHANGELOG.md), under the unreleased version at the top.
+
+Example of changes that should have changelog:
+- Any database change for a stable game
+- Any GUI change (that isn't hidden by `preview mode` or belongs to an experimental game).
+- Changes to multiworld, presets, generator or solver that causes visible changes in the result
```

Comment:
- patcher changes affecting gameplay
- bugfixes and other stability/performance improvements